docsets/Heron/docset.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Heron",
+    "version": "0.14.0",
+    "archive": "Heron.tgz",
+    "author": {
+        "name": "Andrew Jorgensen",
+        "link": "https://twitter.com/ajorgensen"
+    },
+    "aliases": [],
+    "specific_versions": [
+        { 
+            "version": "0.14.0",
+            "archive": "versions/0.14.0/heron.tgz"
```

Comment:
The archive you added is called `Heron.tgz`, not `heron.tgz` (case sensitivity).